docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Sanitize databases
+description: Remove sensitive information from databases on non-production environments to control access.
+---
+
+When working on a new feature on your website, you want to use a new branch.
+Using a new branch makes sure that you don't risk breaking your live, production website.
+
+Creating a branch on Platform.sh copies both the code and the database to that new development branch.
+These code and database changes need to be tested before being merged into production.
+Depending on your processes, internal or external teams may interact with the development environment branch.
+
+Databases of live websites often contain personally identifiable information (PII)
+such as full names, mailing addresses, and phone numbers.
+To ensure people reviewing the code changes can't access information they shouldn't, sanitize the database of any PII that it may contain.
+
+## Before you begin
+
+You need:
+
+- A project with a [MySQL database](../configuration/services/mysql/_index.md).
+- if you aren't using Drupal, you will need the [Platform CLI](/development/cli/_index.md#cli-command-line-interface) installed locally.
+- if you are using Drupal, you will need [Drush](https://www.drush.org/latest/install/) installed.
```

Comment:
Maybe some something like:
```suggestion
- A command interface installed:
  - With Drupal: [Drush](https://www.drush.org/latest/install/)
  - Without Drupal: the [Platform CLI](/development/cli/_index.md#cli-command-line-interface)
```
Not 100% necessary, just seems shorter so maybe clearer?

At the very least, without the future tense and no periods at the end of the list items.